src/figuresClasses.ts

```diff
@@ -1,11 +1,74 @@
-export interface Figure {}
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: 'triangle' = 'triangle';
+
+  constructor(
+    public color: 'red' | 'green' | 'blue',
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (this.a <= 0 || this.b <= 0 || this.c <= 0) {
+      throw new Error(`These sides can not be a less than zero`);
+    }
+
+    if (this.a + this.b <= this.c) {
+      throw new Error(
+        `The sides ${this.a} and ${this.b} can not be less ${this.c}`,
```

Comment:
The error message `The sides ${this.a} and ${this.b} can not be less ${this.c}` is unclear. It should specify that the sum of any two sides must be greater than the third side, e.g., `The sum of sides ${this.a} and ${this.b} must be greater than ${this.c}.`